### PR TITLE
Feature/ui private notes

### PR DESCRIPTION
**Pull request by Rudeka**
Add UI for private notes
<a href='#crh-start'></a><a href='#crh-data-%7B%22processed%22%3A%20%5B%22https%3A//github.com/ITsvetkoFF/Kv-013/pull/122%23discussion_r61492164%22%2C%20%22https%3A//github.com/ITsvetkoFF/Kv-013/pull/122%23discussion_r61493482%22%2C%20%22https%3A//github.com/ITsvetkoFF/Kv-013/pull/122%23discussion_r61493546%22%2C%20%22https%3A//github.com/ITsvetkoFF/Kv-013/pull/122%23discussion_r61493881%22%5D%2C%20%22comments%22%3A%20%7B%22Pull%202749512d57ae9b7b0501db2f60ff23a382dc9602%20DevelopGitHub/GitHubExtension/GitHubExtension.EntryPoint/FrontEnd/src/client/app/roles/roles.html%2032%22%3A%20%7B%22html_url%22%3A%20%22https%3A//github.com/ITsvetkoFF/Kv-013/pull/122%23discussion_r61492164%22%2C%20%22comments%22%3A%20%5B%7B%22body%22%3A%20%22string%20localization%3F%22%2C%20%22created_at%22%3A%20%222016-04-28T19%3A55%3A25Z%22%2C%20%22user%22%3A%20%7B%22avatar_url%22%3A%20%22https%3A//avatars.githubusercontent.com/u/7273003%3Fv%3D3%22%2C%20%22html_url%22%3A%20%22https%3A//github.com/ITsvetkoFF%22%7D%7D%5D%2C%20%22title%22%3A%20%22File%3A%20DevelopGitHub/GitHubExtension/GitHubExtension.EntryPoint/FrontEnd/src/client/app/roles/roles.html%3AL3-42%22%7D%2C%20%22Pull%202749512d57ae9b7b0501db2f60ff23a382dc9602%20DevelopGitHub/GitHubExtension/GitHubExtension.EntryPoint/FrontEnd/src/client/app/roles/roles.html%2033%22%3A%20%7B%22html_url%22%3A%20%22https%3A//github.com/ITsvetkoFF/Kv-013/pull/122%23discussion_r61493546%22%2C%20%22comments%22%3A%20%5B%7B%22body%22%3A%20%22and%20here%5Cr%5Cn%22%2C%20%22created_at%22%3A%20%222016-04-28T20%3A04%3A49Z%22%2C%20%22user%22%3A%20%7B%22avatar_url%22%3A%20%22https%3A//avatars.githubusercontent.com/u/7273003%3Fv%3D3%22%2C%20%22html_url%22%3A%20%22https%3A//github.com/ITsvetkoFF%22%7D%7D%5D%2C%20%22title%22%3A%20%22File%3A%20DevelopGitHub/GitHubExtension/GitHubExtension.EntryPoint/FrontEnd/src/client/app/roles/roles.html%3AL3-42%22%7D%2C%20%22Pull%202749512d57ae9b7b0501db2f60ff23a382dc9602%20DevelopGitHub/GitHubExtension/GitHubExtension.EntryPoint/FrontEnd/src/client/app/roles/roles.controller.js%2017%22%3A%20%7B%22html_url%22%3A%20%22https%3A//github.com/ITsvetkoFF/Kv-013/pull/122%23discussion_r61493482%22%2C%20%22comments%22%3A%20%5B%7B%22body%22%3A%20%22localization%22%2C%20%22created_at%22%3A%20%222016-04-28T20%3A04%3A25Z%22%2C%20%22user%22%3A%20%7B%22avatar_url%22%3A%20%22https%3A//avatars.githubusercontent.com/u/7273003%3Fv%3D3%22%2C%20%22html_url%22%3A%20%22https%3A//github.com/ITsvetkoFF%22%7D%7D%5D%2C%20%22title%22%3A%20%22File%3A%20DevelopGitHub/GitHubExtension/GitHubExtension.EntryPoint/FrontEnd/src/client/app/roles/roles.controller.js%3AL39-65%22%7D%2C%20%22Pull%202749512d57ae9b7b0501db2f60ff23a382dc9602%20DevelopGitHub/GitHubExtension/GitHubExtension.EntryPoint/FrontEnd/src/client/app/roles/roles.html%2010%22%3A%20%7B%22html_url%22%3A%20%22https%3A//github.com/ITsvetkoFF/Kv-013/pull/122%23discussion_r61493881%22%2C%20%22comments%22%3A%20%5B%7B%22body%22%3A%20%22and%20here%22%2C%20%22created_at%22%3A%20%222016-04-28T20%3A07%3A14Z%22%2C%20%22user%22%3A%20%7B%22avatar_url%22%3A%20%22https%3A//avatars.githubusercontent.com/u/7273003%3Fv%3D3%22%2C%20%22html_url%22%3A%20%22https%3A//github.com/ITsvetkoFF%22%7D%7D%5D%2C%20%22title%22%3A%20%22File%3A%20DevelopGitHub/GitHubExtension/GitHubExtension.EntryPoint/FrontEnd/src/client/app/roles/roles.html%3AL3-42%22%7D%7D%7D'></a>
<a href='https://www.codereviewhub.com/'></a>
- [x] <a href='#crh-comment-Pull 2749512d57ae9b7b0501db2f60ff23a382dc9602 DevelopGitHub/GitHubExtension/GitHubExtension.EntryPoint/FrontEnd/src/client/app/roles/roles.html 32'></a> &nbsp;<b><a href='https://github.com/ITsvetkoFF/Kv-013/pull/122#discussion_r61492164'>File: DevelopGitHub/GitHubExtension/GitHubExtension.EntryPoint/FrontEnd/src/client/app/roles/roles.html:L3-42</a></b>
- <a href='https://github.com/ITsvetkoFF'></a> string localization?
- [x] <a href='#crh-comment-Pull 2749512d57ae9b7b0501db2f60ff23a382dc9602 DevelopGitHub/GitHubExtension/GitHubExtension.EntryPoint/FrontEnd/src/client/app/roles/roles.controller.js 17'></a> &nbsp;<b><a href='https://github.com/ITsvetkoFF/Kv-013/pull/122#discussion_r61493482'>File: DevelopGitHub/GitHubExtension/GitHubExtension.EntryPoint/FrontEnd/src/client/app/roles/roles.controller.js:L39-65</a></b>
- <a href='https://github.com/ITsvetkoFF'></a> localization
- [x] <a href='#crh-comment-Pull 2749512d57ae9b7b0501db2f60ff23a382dc9602 DevelopGitHub/GitHubExtension/GitHubExtension.EntryPoint/FrontEnd/src/client/app/roles/roles.html 33'></a> &nbsp;<b><a href='https://github.com/ITsvetkoFF/Kv-013/pull/122#discussion_r61493546'>File: DevelopGitHub/GitHubExtension/GitHubExtension.EntryPoint/FrontEnd/src/client/app/roles/roles.html:L3-42</a></b>
- <a href='https://github.com/ITsvetkoFF'></a> and here
- [x] <a href='#crh-comment-Pull 2749512d57ae9b7b0501db2f60ff23a382dc9602 DevelopGitHub/GitHubExtension/GitHubExtension.EntryPoint/FrontEnd/src/client/app/roles/roles.html 10'></a> &nbsp;<b><a href='https://github.com/ITsvetkoFF/Kv-013/pull/122#discussion_r61493881'>File: DevelopGitHub/GitHubExtension/GitHubExtension.EntryPoint/FrontEnd/src/client/app/roles/roles.html:L3-42</a></b>
- <a href='https://github.com/ITsvetkoFF'></a> and here

<a href='https://www.codereviewhub.com/ITsvetkoFF/Kv-013/pull/122?mark_as_completed=1'></a>&nbsp;<a href='https://www.codereviewhub.com/ITsvetkoFF/Kv-013/pull/122?approve=1'></a>&nbsp;<a href='https://github.com/ITsvetkoFF/Kv-013/pull/122'></a>
<a href='#crh-end'></a>

### CODE REVIEW

**Comment by ITsvetkoFF** [comment]
This pull request is now managed with CodeReviewHub. <a href='https://github.com/ITsvetkoFF/Kv-013/pull/122'>Refresh</a>.<a href='#crh-update-none'></a>

File: DevelopGitHub/GitHubExtension/GitHubExtension.EntryPoint/FrontEnd/src/client/app/roles/roles.html [hunk 1]
```diff
@@ -3,9 +3,39 @@
...
+                                </select>
+                            </td>
+                            <td>
+                                <form name="noteForm" ng-init="collaborator.note"
+                                      ng-submit="noteForm.$valid && vmRoles.createPrivateNote(collaborator)">
+                                    <input type="text" name="newNote" ng-model="collaborator.note" required
+                                           ng-minlength="5">
+                                    <button type="submit" class="btn btn-primary btn-large"
+                                            ng-click="noteForm.newNote.$error.minlength
+                                            && vmRoles.formError('Length must be greater than 5 characters')">
```
Comment (ITsvetkoFF) [inline DevelopGitHub/GitHubExtension/GitHubExtension.EntryPoint/FrontEnd/src/client/app/roles/roles.html]: string localization?

File: DevelopGitHub/GitHubExtension/GitHubExtension.EntryPoint/FrontEnd/src/client/app/roles/roles.controller.js [hunk 2]
```diff
@@ -39,6 +39,26 @@
...
+        vm.createPrivateNote = function(collaborator, note) {
+            githubCollaborators.createPrivateNote(collaborator, note).then(onNoteCreated, onNoteError);
+        };
+
+        vm.formError = function (reason) {
+            logger.error(reason);
+        };
+
+        function onNoteCreated() {
+            logger.info('Note created');
```
Comment (ITsvetkoFF) [inline DevelopGitHub/GitHubExtension/GitHubExtension.EntryPoint/FrontEnd/src/client/app/roles/roles.controller.js]: localization

File: DevelopGitHub/GitHubExtension/GitHubExtension.EntryPoint/FrontEnd/src/client/app/roles/roles.html [hunk 3]
```diff
@@ -3,9 +3,39 @@
...
+                                        Add New Note
```
Comment (ITsvetkoFF) [inline DevelopGitHub/GitHubExtension/GitHubExtension.EntryPoint/FrontEnd/src/client/app/roles/roles.html]: and here

File: DevelopGitHub/GitHubExtension/GitHubExtension.EntryPoint/FrontEnd/src/client/app/roles/roles.html [hunk 4]
```diff
@@ -3,9 +3,39 @@
         <div class="container">
             <div class="row">
                 <div class="col-md-12">
-                    <h1>{{message}}</h1>
-                    <div>{{error}}</div>
-                    <div ng-include="'FrontEnd/src/client/app/roles/collaboratorsdetails.html'" ng-show="!!vmRoles.repo"></div>
+                    <table class="table table-condensed">
+                        <thead>
+                        <tr>
+                            <th>Login</th>
```
Comment (ITsvetkoFF) [inline DevelopGitHub/GitHubExtension/GitHubExtension.EntryPoint/FrontEnd/src/client/app/roles/roles.html]: and here

File: DevelopGitHub/GitHubExtension/GitHubExtension.EntryPoint/FrontEnd/src/client/app/roles/roles.html (hunk 1 above)
Comment (Rudeka) [inline DevelopGitHub/GitHubExtension/GitHubExtension.EntryPoint/FrontEnd/src/client/app/roles/roles.html]: localization added and commited

File: DevelopGitHub/GitHubExtension/GitHubExtension.EntryPoint/FrontEnd/src/client/app/roles/roles.controller.js (hunk 2 above)
Comment (Rudeka) [inline DevelopGitHub/GitHubExtension/GitHubExtension.EntryPoint/FrontEnd/src/client/app/roles/roles.controller.js]: localization added and commited

File: DevelopGitHub/GitHubExtension/GitHubExtension.EntryPoint/FrontEnd/src/client/app/roles/roles.html (hunk 3 above)
Comment (Rudeka) [inline DevelopGitHub/GitHubExtension/GitHubExtension.EntryPoint/FrontEnd/src/client/app/roles/roles.html]: localization added and commited

File: DevelopGitHub/GitHubExtension/GitHubExtension.EntryPoint/FrontEnd/src/client/app/roles/roles.html (hunk 4 above)
Comment (Rudeka) [inline DevelopGitHub/GitHubExtension/GitHubExtension.EntryPoint/FrontEnd/src/client/app/roles/roles.html]: localization added and commited

**Pull request merged**